src/components/Card/Toolbar.tsx

```diff
@@ -0,0 +1,8 @@
+import { Box, Sprinkles } from "@saleor/macaw-ui-next";
+import React from "react";
+
+export const Toolbar: React.FC<Sprinkles> = ({ children }) => (
+  <Box position="absolute" right={6} top={6} display="flex" flexDirection="row" gap={2}>
+    {children}
+  </Box>
+);
```

Comment:
question: Is it possible that Toolbar cover Title or other content?

Reply:
I changed it so that it won't